zerver/openapi/python_examples.py

```diff
@@ -358,7 +358,7 @@ def list_subscriptions(client):
     # type: (Client) -> None
     # {code_example|start}
     # Get all streams that the user is subscribed to
-    result = client.list_subscriptions()
+    result = client.list_subscriptions(request={"include_subscribers": True})
```

Comment:
We probably don't want to provide this argument in the example, since we expect most users of this endpoint won't want the subscriber data.